### PR TITLE
OCPBUGS-43034: Add Route Filter in route list

### PR DESCRIPTION
**Pull request by upalatucci**
Screenshot 2024-10-14 at 11 47 42
Screenshot 2024-10-14 at 11 47 46
Screenshot 2024-10-14 at 11 47 50
Screenshot 2024-10-14 at 11 47 53

### CODE REVIEW

File: src/utils/constants/routes.ts
Comment (upalatucci) [inline src/utils/constants/routes.ts:1]: Const to use for the `map` in `useRouteFilters`
and type to use for Typescript return type

File: src/views/routes/list/utils/utils.ts [hunk 1]
```diff
@@ -1,9 +1,10 @@
 import * as _ from 'lodash';
```
Comment (metalice) [inline src/views/routes/list/utils/utils.ts:1]: can we remove lodash?

Comment (upalatucci) [reply src/views/routes/list/utils/utils.ts:1]: We can do that in a different pr

**Pull request merged**